### ISSUE TITLE
Is there an FPS cap?

### ISSUE THREAD

**Issue by lmills87** [RobertBeckebans/RBDOOM-3-BFG#456]
I've been searching google but am having a hard time finding a definite answer on this. I have Doom 3 (non-BFG) and it seems to have a 60 FPS cap that can't be bypassed by disabling V-Sync.

Is BFG the same way, or does this source port allow for greater than 60 FPS without otherwise breaking the game?

**Comment by RobertBeckebans** [comment]
Yes there is the console variable com_engineHz which can be either 60 or 120 in the BFG edition. It's offered in the video options. In regular D3 it's somewhat bound to the sound decoder thread which is set to work with 60 fps.

**Comment by lmills87** [comment]
> Yes there is the console variable com_engineHz which can be either 60 or 120 in the BFG edition. It's offered in the video options. In regular D3 it's somewhat bound to the sound decoder thread which is set to work with 60 fps.

Thank you! If this is getting too off topic let me know. I’m trying to determine the “best” way to replay D3. I’m interested in high frame rate and improved textures/effects while maintaining a mostly vanilla experience.

I’ve heard that FPS can be unlocked in Dhewm as well but what considerations might be made when comparing your project to Dhewm with regards to playability and the above mentioned things? Is higher FPS, texture mods and the like handles better in RBDoom-3-BFG? As far as you’re concerned is there any reason why vanilla Doom + mods or Dhewm might be desired over RBDOOM? Your project seems the most appealing to me compared to the other options thus far, but I’m sure I’m not seeing the whole picture as a lay person.

**Comment by DanielGibson** [comment]
unlocking the framerate in dhewm3 is buggy

**Comment by lmills87** [comment]
Another question if I may. In going through the install, I created a new folder (Doom 3 BFG) and copied the entire base folder from installed Steam BFG to my new folder. Then I copied RBDoom3BFG.exe to my new folder. So all I have is the .exe, and the base folder with all its contents.

The game seems to launch. However, I noticed in the instructions that it mentions FFmpeg DLLs as well, but I did not see a description of what these are. What are these files and do I need them?
